backend/uasmartsignage/pom.xml

```diff
@@ -15,6 +15,8 @@
 	<description>Projeto de Informática</description>
 	<properties>
 		<java.version>17</java.version>
+		<sonar.organization>ua-smart-signage-platform</sonar.organization>
+		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
```

Comment:
Is this needed when it is mentioned in the properties?

Reply:
This is necessary, without it does not work.